lambdas/check-coi/src/main/java/uk/gov/di/ipv/core/checkcoi/CheckCoiHandler.java

```diff
@@ -98,26 +113,54 @@ public Map<String, Object> handleRequest(ProcessRequest request, Context context
                             .toList();
 
             var coiSubjourneyType = ipvSession.getCoiSubjourneyType();
-            var successfulCheck =
+            var coiCheck =
```

Comment:
Using CoiCheck because we need to know both the success & type of check made in order to make the audit event